Docs/Debugging/Debugging with Eclipse and pyOCD.md

```diff
@@ -0,0 +1,68 @@
+#Debugging with Eclipse and pyOCD
+
+This article explains how to debug an exported mbed project with Eclipse and pyOCD. To debug a project, you need to first export it from mbed as a makefile project, import the makefile project into Eclipse and finally setup debugger configurations. Below are these steps but in much more detail.
```

Comment:
I don't like "need to first". It's a confusing sentence if you're not a native speaker.
Also, Makefile. 
Set up (verb is two words).
I also don't like "Below are these steps but in much more detail". It's very clumsy.